### ISSUE TITLE
WebApp seems to be broken

### ISSUE THREAD

**Issue by paresy** [skaut/shared-drive-mover#920]
When opening your Demo URL (https://presun-disku.skauting.cz/) i only get the following error:

![image](https://user-images.githubusercontent.com/260376/166901203-bd164d4d-9985-4967-8b5d-fa9c270e6db4.png)

I also tried in incognito mode to make sure that i do not run in any cookie/login problems.

When deploying the app myself everything works as long as i only allow logins from within my G Suite domain. When allowing access from anyone with a Google account i receive the mentioned error.

Do you have any idea what might be wrong?

Thanks! (This tool might be the hero we desperately need to migration to shared drives...)

**Comment by marekdedic** [comment]
Hi,
the demo at https://presun-disku.skauting.cz/ is only available for users in the @skaut.cz domain. To run this yourself, you have to make your own deployment - see the readme.

**Comment by paresy** [comment]
Hi Marek,

and thanks for the super fast reply. That's what i did and as long as the deployment is limited to "GSuite Team" users everything works file. Though I would like to move data from a public gmail user to a team drive and thought that changing access to "everyone with a google account" would allow the app to operate in this scope. (Btw: Using you link does not ask me for any permission, it just fails with that error)

Unfortunately i get the error message above. Would you mind giving me a hint how to debug this error? Or is it generally not possible to achieve that i want?

(The team drive is properly shared for the person outside of the gsuite domain)

Thanks again!

**Comment by paresy** [comment]
I changed the following option in the manifest and redeployed the app. Now it works. 

![image](https://user-images.githubusercontent.com/260376/166910334-4d1c12a8-fcaf-4bd4-8606-b287e1f632a6.png)

Seems like the app also got confused when having multiple accounts. I opened the URL that the App Script deployment provided directly inside an incognito tab which worked. When opening with a logged in user, that URL gets redirect and that redirection link does not work properly inside an incognito tab. See here: https://stackoverflow.com/a/47050007/10288655

Thanks!
